modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ClearCachesTaskArg.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2024 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache;
```

Comment:
It would be nice to move this class (`ClearCachesTask` and `ClearCachesTaskResult` as well) to the `org.apache.ignite.internal.visor.cache` package.

Also, please add these classes to the `classnames.properties` file https://github.com/gridgain/gridgain/blob/master/modules/core/src/main/resources/META-INF/classnames.properties

Reply:
done